SOURCES/openvswitch-2.17.7-comment-failing-tests.XCP-ng.patch

```diff
@@ -1,3 +1,5 @@
+Some tests fail sometimes
```

Comment:
Here maybe write the list of disabled tests, why each is disabled, and whether the failure is each time or only some times?